Intermittent test failure for `TestREClientIntegration.test_admin_empty_query`

I'm consistently seeing these four tests fail every other run:
```
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_empty_query
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_missing_param
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_query_empty_auth
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_query_invalid_auth
```

Although I've seen these fail once:
```
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_empty_query
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_missing_param
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_query_empty_auth
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_query_invalid_auth
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_query_ok
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_admin_raise_not_found
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_save_docs_empty_auth
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_save_docs_invalid_auth
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_save_docs_invalid_docs
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_save_docs_ok
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_save_docs_unknown_coll
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_stored_query_missing_bind_vars
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_stored_query_ok
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_stored_query_raise_not_found
FAILED client_src/test/test_integration.py::TestREClientIntegration::test_stored_query_unknown_query
```

It seems like it repeatably fails every other test, suggesting that the passing tests are leaving state that breaks the next run, and the failing tests somehow clear it up

--- a/README.md
+++ b/README.md
@@ -49,3 +49,10 @@ The CI service is available in the `relationapi` service
 * Merge from Develop to Main
 * Draft a release using semantic versioning (e.g 1.2.3) by creating a tag with the target of Main
 * Ensure release notes are updated manually and/or by AutoGenerate Release Notes
+
+## Testing
+
+To run tests:
+```
+make test
+```

--- a/scripts/run_tests.sh
+++ b/scripts/run_tests.sh
@@ -16,8 +16,10 @@ sh /app/scripts/start_server.sh &
 coverage erase
 # spec validation
 python -m spec.validate
+# wait for the RE service to be up so integration tests can pass
+python -m relation_engine_server.utils.wait_for api
 # run importer/, relation_engine_server/, and spec/ tests
 coverage run --branch -m pytest
-# RE client tests
+# RE client tests - seemss like these tests were already run in the line above, why run them again?
 PYTHONPATH=client_src python -m pytest client_src/test
 coverage html --omit=*/test_*

